src/ui/public/notify/partials/toaster.html

```diff
@@ -48,6 +46,14 @@
             ng-class="'btn-' + notif.type"
             ng-click="notif.address()"
             >Fix it</button>
+          <button
+            type="button"
+            class="btn"
+            ng-repeat="action in notif.customActions"
+            ng-class="'btn-' + notif.type"
```

Comment:
This may be beyond the scope of this PR, but I think it will need to be done someday: I think we should be more explicit in our class assignment, e.g. via methods like `getButtonClass(notif.type)` and `getToastClass(notif.type)`. This will uncouple the type from the class names for these elements, which will let us remove the 'error'/'danger' substitution on line 431. This will also give us to change the implementation details of these different types without having to change the type values in the future.